AK/NonnullPtrVector.h

```diff
@@ -26,10 +26,14 @@ class NonnullPtrVector : public Vector<PtrType, inline_capacity> {
         : Base(static_cast<Base const&>(other))
     {
     }
+    NonnullPtrVector(std::initializer_list<PtrType> list)
+        : Base(list)
+    {
+    }
 
     using Base::size;
 
-    using ConstIterator = SimpleIterator<const NonnullPtrVector, const T>;
+    using ConstIterator = SimpleIterator<const NonnullPtrVector, T const>;
```

Comment:
clang-format why are you like this

Reply:
clang-format-15 was just released, I wonder if there are any improvements

Reply:
yak-format wen?